### ISSUE TITLE
Ubuntu 21.10 (impish) support

### ISSUE THREAD

**Issue by lenybernard** [dvorka/hstr#436]
Hello,

I try to install hstr on a fresh new ubuntu 21.10 but there isn't any release :
```
E: The repository 'http://ppa.launchpad.net/ultradvorka/ppa/ubuntu impish Release' does not have a Release file.
```
Can I do something ?

Thank you

**Comment by zvezdochiot** [comment]
Hi @lenybernard .

https://pkgs.org/search/?q=hstr

Resolved.

**Comment by dvorka** [comment]
@lenybernard @zvezdochiot thank you! `impish` distribution is now available in my PPA as well: https://launchpad.net/~ultradvorka/+archive/ubuntu/ppa/+packages

**Comment by gitressa** [comment]
This looks great thanks. I could make a new issue, but I just wanted to comment that the Ubuntu 21.04 link in the README file doesn't work: https://github.com/dvorka/hstr/blob/master/INSTALLATION.md#ubuntu

> HSTR is [included in Ubuntu 21.10](https://packages.ubuntu.com/impish/utils/hstr) and [newer](https://wiki.ubuntu.com/Releases) releases:

- Dead link: https://packages.ubuntu.com/impish/utils/hstr
- Maybe better to use this? https://packages.ubuntu.com/search?keywords=hstr
